diff -Naur xfce4-panel-4.11.1/plugins/systray/Makefile.am xfce4-panel-4.11.1.tpg/plugins/systray/Makefile.am
--- xfce4-panel-4.11.1/plugins/systray/Makefile.am	2014-05-02 19:54:12.000000000 +0000
+++ xfce4-panel-4.11.1.tpg/plugins/systray/Makefile.am	2014-09-20 12:59:47.691959060 +0000
@@ -49,7 +49,8 @@
 	$(EXO_LIBS) \
 	$(LIBXFCE4UTIL_LIBS) \
 	$(LIBXFCE4UI_LIBS) \
-	$(XFCONF_LIBS)
+	$(XFCONF_LIBS) \
+	-lm
 
 libsystray_la_DEPENDENCIES = \
 	$(top_builddir)/libxfce4panel/libxfce4panel-$(LIBXFCE4PANEL_VERSION_API).la \
